fhir/com.b2international.snowowl.fhir.core/src/org/linuxforhealth/fhir/model/r5/type/code/FHIRVersion.java

```diff
@@ -0,0 +1,1232 @@
+
+    /**
+     * 4.1.0
+     * 
+     * <p>R4B Ballot #1 + Connectathon 27 (Virtual).
+     */
+    public static final FHIRVersion VERSION_4_1_0 = FHIRVersion.builder().value(Value.VERSION_4_1_0).build();
+
+    /**
+     * 4.2
+     * 
+     * <p>R5 Preview #1 version.
+     */
+    public static final FHIRVersion VERSION_4_2 = FHIRVersion.builder().value(Value.VERSION_4_2).build();
+
+    /**
+     * 4.2.0
+     * 
+     * <p>R5 Preview #1 + Connectathon 23 (Sydney).
+     */
+    public static final FHIRVersion VERSION_4_2_0 = FHIRVersion.builder().value(Value.VERSION_4_2_0).build();
+
+    /**
+     * 4.3
+     * 
+     * <p>R4B version.
+     */
+    public static final FHIRVersion VERSION_4_3 = FHIRVersion.builder().value(Value.VERSION_4_3).build();
+
+    /**
+     * 4.3.0
+     * 
+     * <p>FHIR Release 4B (Normative + STU).
+     */
+    public static final FHIRVersion VERSION_4_3_0 = FHIRVersion.builder().value(Value.VERSION_4_3_0).build();
+
+    /**
+     * 4.3.0-cibuild
+     * 
+     * <p>FHIR Release 4B CI-Builld.
+     */
+    public static final FHIRVersion VERSION_4_3_0_CIBUILD = FHIRVersion.builder().value(Value.VERSION_4_3_0_CIBUILD).build();
+
+    /**
+     * 4.3.0-snapshot1
+     * 
+     * <p>FHIR Release 4B Snapshot #1.
+     */
+    public static final FHIRVersion VERSION_4_3_0_SNAPSHOT1 = FHIRVersion.builder().value(Value.VERSION_4_3_0_SNAPSHOT1).build();
+
+    /**
+     * 4.4
+     * 
+     * <p>R5 Preview #2 version.
+     */
+    public static final FHIRVersion VERSION_4_4 = FHIRVersion.builder().value(Value.VERSION_4_4).build();
+
+    /**
+     * 4.4.0
+     * 
+     * <p>R5 Preview #2 + Connectathon 24 (Virtual).
+     */
+    public static final FHIRVersion VERSION_4_4_0 = FHIRVersion.builder().value(Value.VERSION_4_4_0).build();
+
+    /**
+     * 4.5
+     * 
+     * <p>R5 Preview #3 version.
+     */
+    public static final FHIRVersion VERSION_4_5 = FHIRVersion.builder().value(Value.VERSION_4_5).build();
+
+    /**
+     * 4.5.0
+     * 
+     * <p>R5 Preview #3 + Connectathon 25 (Virtual).
+     */
+    public static final FHIRVersion VERSION_4_5_0 = FHIRVersion.builder().value(Value.VERSION_4_5_0).build();
+
+    /**
+     * 4.6
+     * 
+     * <p>R5 Draft Ballot version.
+     */
+    public static final FHIRVersion VERSION_4_6 = FHIRVersion.builder().value(Value.VERSION_4_6).build();
+
+    /**
+     * 4.6.0
+     * 
+     * <p>R5 Draft Ballot + Connectathon 27 (Virtual).
+     */
+    public static final FHIRVersion VERSION_4_6_0 = FHIRVersion.builder().value(Value.VERSION_4_6_0).build();
+
+    /**
+     * 5.0
+     * 
+     * <p>R5 Versions.
+     */
+    public static final FHIRVersion VERSION_5_0 = FHIRVersion.builder().value(Value.VERSION_5_0).build();
+
+    /**
+     * 5.0.0
+     * 
+     * <p>R5 Final Version.
+     */
+    public static final FHIRVersion VERSION_5_0_0 = FHIRVersion.builder().value(Value.VERSION_5_0_0).build();
+
+    /**
```

Comment:
Do we need new interim/snapshot/earlier than 4.0.1 versions? I would remove them and support only the 4.0.1 -> 5.0.0 version range.